index.js

```diff
@@ -235,6 +254,12 @@ BabelClient.prototype.createAnnotation = function(token, data, callback){
         }
     };
 
+    // if options.xIngestSynchronously set to true, then and only then add the header
+    // otherwise leave it out which defaults to false
+    if(options && options.xIngestSynchronously && options.xIngestSynchronously === true ){
```

Comment:
just a thought - but could options be used for anything other than headers? Should it be ```options.headers.whatevs```

Also - should it support ```options.headers['X-Ingest-Synchronously']``` rather than the camelCased version?